sdk/nextjs/src/server/DevCycleClientProvider.tsx

```diff
@@ -0,0 +1,71 @@
+import 'server-only'
+import {
+    getClient,
+    getDevCycleServerData,
+    identifyInitialUser,
+    identifyUser,
+    setClient,
+    setSDKKey,
+} from '@devcycle/next-sdk/server'
+import React from 'react'
+import { DevCycleUser, initializeDevCycle } from '@devcycle/js-client-sdk'
+import { DevCycleClientProviderClientSide } from '../client/DevCycleClientProviderClientside'
+import { getDVCCookie } from './cookie'
+
+type DevCycleServerOptions = {
+    initialUserOnly?: boolean
+}
+
+type DevCycleClientProviderProps = {
+    sdkKey: string
+    user: DevCycleUser
+    options?: DevCycleServerOptions
+    children: React.ReactNode
+}
+
+export const initialize = async (
```

Comment:
function called on every server render pass to get bucketed config for current user.